apps/demos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts

```diff
@@ -148,6 +171,13 @@ export class AppComponent {
     const endDateEditor = form.getEditor('endDate');
     endDateEditor.option('disabledDates', holidays);
   };
+
+  setComponentAria(element): void {
+    const prevAria = element?.attr('aria-label') || '';
+    element?.attr({
+      'aria-label': [prevAria, this.ariaDescription()].filter(Boolean).join(', '),
```

Comment:
There is an easier option:
`element?.attr('aria-label', ${prevAria} ${this.ariaDescription()});`